src/util/types.ts

```diff
@@ -1266,11 +1266,13 @@ export const adaptEventsOfChild = (
   return out;
 };
 
+export type TooltipEventType = 'axis' | 'item';
```

Comment:
Nice! I wonder whether we can make this an actual option to the user instead of having it be hard defined per chart type. 

For example in the case of the multi line chart, it is a standard use case to only want to show a single dots Tooltip (as opposed to recharts default behaviour of showing all values per xAxis value). 

If I understand correctly this is the switch between the two behaviours.

Reply:
Totally unrelated to this PR, just wanted to share some more context knowledge.